easybuild/tools/modules.py

```diff
@@ -1385,14 +1385,14 @@ def invalidate_module_caches_for(path):
     if not os.path.exists(path):
         raise EasyBuildError("Non-existing path specified to invalidate module caches: %s", path)
 
-    _log.debug("Invallidating module cache entries for path '%s'", path)
+    _log.debug("Invalidating module cache entries for path: %s", path)
```

Comment:
Parenthesis are missing here

Reply:
parentheses are not needed here, because the path is at the end of the string.